docs/usage/attributes.md

```diff
@@ -93,13 +93,15 @@ Cloud providers with supported metadata endpoints and their respective attribute
   - AFTERBURN_OPENSTACK_HOSTNAME
   - AFTERBURN_OPENSTACK_IPV4_LOCAL
   - AFTERBURN_OPENSTACK_IPV4_PUBLIC
-  - AFTERBURN_OPENSTACK_INSTANCE_ID
+  - AFTERBURN_OPENSTACK_INSTANCE_ID (from the EC2 metadata)
```

Comment:
hmm wdyt? 

AFTERBURN_OPENSTACK_INSTANCE_ID _EC2 metadata_
AFTERBURN_OPENSTACK_INSTANCE_UUID _Nova metadata_

Reply:
Or is it more valuable to the end user to say which one is better for which version of openstack? 

IE. when ID should be used to fetch data from the EC2 or what version and newer should use the newer attribute?  x.y.z >= nova and x.y.z <= ec2

Reply:
As far as I can tell, there are exactly zero valid reasons to use the ID from the EC2 metadata, if the Nova UUID is available. I think that your first proposal makes most sense

Reply:
Apologies for the continued bikeshedding, but this feels a bit too brief for users who may not understand the distinction.  What about something like:

> - AFTERBURN_OPENSTACK_INSTANCE_ID &mdash; EC2-style instance ID, not used in OpenStack API
> - AFTERBURN_OPENSTACK_INSTANCE_UUID &mdash; OpenStack instance UUID (Afterburn &ge; 5.5.0)